### PR TITLE
sadique | Design changes in template to add composition scheme text b…

### PR DESCRIPTION
**Pull request by sadik-walkover**
…ox both in HTML and PDF file

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-a/gst-template-a.component.scss [hunk 1]
```diff
@@ -513,3 +513,9 @@
     height: 60px;
     min-width: 40px;
   }
+  .header-table {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-a/gst-template-a.component.scss:516]: run formatting

File: apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.html [hunk 2]
```diff
@@ -25,6 +25,14 @@ <h2>Header</h2>
                 <div *ngIf="fieldsAndVisibility.header.gstin" class="form-group">
                     <input type="checkbox" [(ngModel)]="customTemplate.sections['header'].data['gstin'].display" (ngModelChange)="onChangeFieldVisibility(null, null, null)" checked /> GSTIN
                 </div>
+                <ng-template>
+                    <div class="form-group clearfix customField">
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.html:29]: use hypen-case class

Comment (sadik-walkover) [reply apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.html:29]: customField this class too many places we we cant change right now

File: apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-a/gst-template-a.component.scss (hunk 1 above)
Comment (sadik-walkover) [reply apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-a/gst-template-a.component.scss]: Done

**Pull request merged**